File: leetcode2024/2024_12/leetcode_3274_checkTwoChessboards.cpp
#include <iostream>
#include <string>
using namespace std;

class Solution {
public:
    bool checkTwoChessboards(string c1, string c2) {
        return ((c1[0]-'a')+(c1[1]-'0'))%2 == ((c2[0]-'a')+(c2[1]-'0'))%2;
    }
};

int main(){
    Solution s;
    string c1 = "a1", c2 = "c3";
    cout << s.checkTwoChessboards(c1, c2) << endl;
}

File: leetcode2024/2024_12/leetcode_3274_checkTwoChessboards.py
class Solution:
    def checkTwoChessboards(self, c1: str, c2: str) -> bool:
        return (ord(c1[0])+int(c1[1]))%2 == (ord(c2[0])+int(c2[1]))%2

if __name__ == "__main__":
    s = Solution()
    c1 = "a1"
    c2 = "c3"
    print(s.checkTwoChessboards(c1, c2))
